src/utils/markets.tsx

```diff
@@ -330,6 +343,28 @@ export function _useUnfilteredTrades(limit = 10000) {
   //   .map(market.parseFillEvent.bind(market));
 }
 
+export function useBonfidaTrades() {
+  const { market } = useMarket();
+  async function getBonfidaTrades() {
+    if (!market) {
+      return null;
+    }
+    const response = await fetch(
+      `https://serum-api.bonfida.com/trades/address/${market.address.toBase58()}`,
+    );
+    if (response.ok) {
+      const responseJson = await response.json();
+      return responseJson.success ? responseJson.data : null;
+    }
+  }
+  const [trades] = useAsyncData(
```

Comment:
We should just return the results of `useAsyncData` here in case some downstream consumer wants to use trades using the loaded flag

Reply:
Done!